values.yaml

```
@@ -72,7 +72,20 @@ taiga:
   # emailSmtpPort: 587
   # emailSmtpUser: "user@example.com"
   # emailSmtpPassword: ""
-
+  ldap:
+    Enabled: False
+    use_tls: False
+    ldap_server: ldap://<ldap_server_host_name>
+    port: 389
+    bind_dn: "cn=something,ou=admins,ou=theouhere,ou=auth,o=ethz,c=ch"
+    base_dn: "basednhere"
+    password: "changeme"
+    username_attribute: "uid"
+    email_attribute: "mail"
+    full_name_attribute: "cn"
+    fallback: "normal"
+    save_password: False
+    filter: "(&(objectClass=posixAccount)(ou=some-ldap-group))"
```

I'd put this to the image default, to minimize user surprise
```suggestion
    filter: "(mail=*)"
```